build.gradle

```diff
@@ -117,6 +120,42 @@ configure(subprojects - project(':integration')) {
             }
         }
     }
+
+    apply plugin: 'maven-publish'
```

Comment:
We’ve discussed this and in the next step we will use only `maven-publish` plugin instead of `com.bmuschko.nexus`, so duplicated `pom` configuration will be eliminated.

Reply:
unfortunately we have to stay with gradle-nexus-plugin and nexus-staging because we use the mechanism of staging repository deploy not direct deploy